src/gluonts/evaluation/metrics.py

```diff
@@ -67,7 +67,7 @@ def abs_error(target: np.ndarray, forecast: np.ndarray) -> float:
     r"""
     .. math::
 
-        abs\_error = sum(|Y - \hat{Y}|)
+    abs\_error = sum(|Y - \hat{Y}|)
```

Comment:
According to [this](https://gitlab.aws.dev/ml-forecasting/research/index-selection-code/-/tree/main/data?ref_type=heads) the indentation is needed

Reply:
good point and I agree, will check why

Reply:
ok, the problem here seems to be missing baby docs before the equation, so the `.. math::` misinterpreted as summary